contracts/interfaces/ITaxManager.sol

```diff
@@ -2,6 +2,16 @@
 pragma solidity ^0.8.0;
 
 interface ITaxManager {
+    struct SeekerFees {
+        uint256 referralRewards;
+        uint256 platformRevenue;
+    }
+
+    function getSeekerFees() external view returns (SeekerFees memory);
+
+    function getReferralTaxReceiver() external view returns (address);
```

Comment:
referral tax goes to referrers and leftovers (40 percent or more) go to platform pool, different with the below one though